### ISSUE TITLE
Differentiating between culled and non culled animals 

### ISSUE THREAD

**Issue by aisha322** [luansheng/visPedigree#3]
Hello! Thank you so much for sharing your code! I'm not sure if this is the right place to ask this, but is there a way to use vispedigree to differentiate between culled and non culled animals? I've played around a bit and am unsure of how to do this. Thank you in advance for all the help!

**Comment by luansheng** [comment]
Can you give me about the detailed parameters about differences between two pedigrees?
&nbsp;sheng luan

---Original---

**Comment by aisha322** [comment]
Absolutely. Thank you for the quick response. Essentially this is a macaque pedigree where I need to differentiate which animals in four generations have been euthanized. I was able to create a pedigree chart that shows the lineage and which animals have children/ are single. In addition to this though, I would like to maybe change the color or shape of the individuals based on a column that states whether or not it has been euthanized. Hopefully this helps. If there is anymore clarification needed feel free to reach out.

**Comment by luansheng** [comment]
Ok. I will try to set the color or shape parameters of individuals.

**Comment by aisha322** [comment]
Thank You! That would be very helpful!

**Comment by luansheng** [comment]
hi, I have added a new parameter of "highlight" to mark the targeted individuals in the visped().
Please update to 0.4.1.
